wordpress-mobile/WordPress-iOS
# Jetpack App (Emphasis): Update Jetpack screenshots

Part of #16395
Follow-on to https://github.com/wordpress-mobile/WordPress-iOS/pull/16396

## Description

This PR updates the JetpackScreenshotGeneration target.
- Added new stubs to WordPressMocks (https://github.com/wordpress-mobile/WordPressMocks/pull/25)
- Updated the My Site screenshot flow to switch to a site with Jetpack Scan / Backup capabilites turned on

Design ref: pcdRpT-n0-p2

## Merge directions

- Merge after https://github.com/wordpress-mobile/WordPressMocks/pull/28

## How to test
1. Switch to the `JetpackScreenshotGeneration` target
2. Run `rake mocks` to start a local mock server for UI tests
3. ✅ Make sure that `testGenerateScreenshots()` succeeds

My Site | Activity Log | Jetpack Scan | Jetpack Backup | Stats
-- | -- | -- | -- | --
![Simulator Screen Shot - iPhone 12 Pro - 2021-05-11 at 15 34 04](https://user-images.githubusercontent.com/6711616/117932572-7a3a1000-b33b-11eb-9a37-6971dff54b4b.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-10 at 15 05 42](https://user-images.githubusercontent.com/6711616/117614338-24326480-b1a3-11eb-842d-623b998e7339.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-10 at 15 05 52](https://user-images.githubusercontent.com/6711616/117614339-24326480-b1a3-11eb-87bc-04a84e437b07.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-10 at 15 06 04](https://user-images.githubusercontent.com/6711616/117614343-25639180-b1a3-11eb-8eeb-1f25be5006b4.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-10 at 15 06 21](https://user-images.githubusercontent.com/6711616/117614345-25fc2800-b1a3-11eb-9001-c57b4c159b9a.png)


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
